docs/source/09_development/02_set_up_pycharm.md

```diff
@@ -122,6 +122,33 @@ Click **OK** and then select **Remote Run** from the toolbar and click **Run** t
 
 To remotely debug, click the debugger button as [described above](#debugging).
 
+
+## Advanced: Docker interpreter
+
+```eval_rst
+.. note::  This section uses features supported in PyCharm Professional Edition only.
+```
+
+Firstly, add a Docker interpreter. Go to **Preferences | Project Interpreter** as above and proceed to add a new interpreter. Select **Docker Interpreter** and select the target Docker image:
+
+![](../meta/images/pycharm_docker_interpreter_1.png)
+
+Click **Next** and check that the required packages appear:
+
+![](../meta/images/pycharm_docker_interpreter_2.png)
+
+Click **Next** and wait for PyCharm to index your new Python interpreter.
+
+Click **OK**. Go to **Run / Debug Configurations** to add a **Python Run**. For the script path, get the path of the Kedro CLI on the remote computer by running `which kedro` (macOS / Linux) in the appropriate environment (/usr/local/bin/kedro in this case). Specify `run` as the parameter. Specify your working directory as the directory that contains your kedro project directory on your local machine. Optional: Edit the Docker container volume mount settings if it does not match the directory that contains your kedro project directory.   
+
+![](../meta/images/pycharm_docker_interpreter_3.png)
+
+Click **OK** and then select your run configuration from the toolbar and click **Run** to execute.
+
+![](../meta/images/pycharm_docker_interpreter_4.png)
```

Comment:
I really like the previous three screenshots, but wonder whether we should keep this one. Wdyt? What additional value do you think it brings?

Reply:
Good point, I removed the last screenshot from the md file and the meta folder.